docs/tutorials/ctapipe_handson.ipynb

```diff
@@ -0,0 +1,659 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "first let's select only those telescopes with LST:LSTCam"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "subarray.telescope_types"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "subarray.get_tel_ids_for_type(\"LST:LSTCam\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's write out program"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = utils.get_dataset(\"gamma_test_large.simtel.gz\") \n",
+    "source = event_source(data, allowed_tels=[1,2,3,4],  max_events=10) # remove the max_events limit to get more stats"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for event in source:\n",
+    "    calib.calibrate(event)\n",
+    "    \n",
+    "    for tel_id, tel_data in event.dl1.tel.items():\n",
+    "        tel = event.inst.subarray.tel[tel_id]\n",
+    "        mask = tailcuts_clean(tel.camera, tel_data.image[0])\n",
+    "        #params = hillas_parameters(tel.camera[mask], tel_data.image[0][mask])\n",
+    "        #print(params)\n",
+    "        params = hillas_parameters(tel.camera[mask], tel_data.image[0][mask])\n",
+    "        #print(params)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ctapipe.io import HDF5TableWriter\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with HDF5TableWriter(filename='hillas.h5', group_name='dl1', overwrite=True) as writer:\n",
+    "    \n",
+    "    for event in source:\n",
+    "        calib.calibrate(event)\n",
+    "    \n",
+    "        for tel_id, tel_data in event.dl1.tel.items():\n",
+    "            tel = event.inst.subarray.tel[tel_id]\n",
+    "            mask = tailcuts_clean(tel.camera, tel_data.image[0])\n",
+    "            params = hillas_parameters(tel.camera[mask], tel_data.image[0][mask])\n",
+    "            writer.write(\"hillas\", params)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### We can now load in the file we created and plot it"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!ls *.h5"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "hillas = pd.read_hdf(\"hillas.h5\", key='/dl1/hillas')\n",
```

Comment:
Assumed, somebody just sent me `hillas.h5` via email .. how would I know the key?

Reply:
Maybe one could show a bit how dataset exploration works in ctapipe

Reply:
yes, i think many of these noteboooks will need improvement, for now I'm just moving them over from the old dir and not making large mods other than bug fixes.

To look inside a hdf5 file there are many options `h5ls` probably is the easiest